formats/opentype.fathom

```diff
@@ -1314,8 +1489,8 @@ let table_directory = fun (file_start : Pos) => {
                     maxp.num_glyphs,
             },
             name <- required_table "name" name_table,
-            os2 <- required_table "OS/2" unknown_table,
-            post <- required_table "post" unknown_table,
+            os2 <- required_table_with_len "OS/2" os2_table,
```

Comment:
@brendanzab I rebased on `main`. I added this new function because `OS/2` needs access to the table length. Not sure if this is the best way or not.

Reply:
Oh cool, yeah I think this is ok for now! 👍